composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "broadway/read-model-mongodb",
+    "description": "A MongoDB ReadModel implementation for Broadway.",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Robin van der Vleuten",
+            "email": "robin@webstronauts.co"
+        }
+    ],
+    "homepage" : "https://github.com/broadway/read-model-mongodb",
+    "autoload": {
+        "psr-4": {
+            "Broadway\\ReadModel\\": "src"
```

Comment:
It's the same namespace as the ES read model implementation. I guess the class name itself is verbose enough to notice that it is the mongodb implementation.